src/server/pfs/server/driver.go

```diff
@@ -147,7 +154,10 @@ func newDriver(address string, etcdAddresses []string, etcdPrefix string, treeCa
 		},
 		openCommits: pfsdb.OpenCommits(etcdClient, etcdPrefix),
 		treeCache:   treeCache,
+		storageRoot: storageRoot,
 	}
+	// Allow up to a third of the requested memory to be used for memory intensive operations
+	memoryLimiter = semaphore.NewWeighted(memoryRequest / 3)
```

Comment:
Updating a global variable from within a constructor without a `sync.Once` seems a bit weird to me.

Reply:
Fair, I can add a once to this even though we generally only have one driver.

Reply:
Yeah, after thinking about it, I think I agree. And yeah, we should only have 1 driver at a time in a container.